src/views/landing.njk

```diff
@@ -7,36 +7,36 @@
   {{ serviceName }}
 {% endblock %}
 
-{% block beforeContent %}
-  {{
-    govukBreadcrumbs({
-      items: [
-        { href: 'https://www.gov.uk/', text: 'Home' },
-        { href: 'https://www.gov.uk/browse/business', text: 'Business and self-employed' },
-        { href: 'https://www.gov.uk/browse/business/limited-company', text: 'Running a limited company' }
-      ]
-    })
-  }}
-{% endblock %}
-
 {% block content %}
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">{{ serviceName }}</h1>
-      <p class="govuk-body-l">Use this service to get your NodeJS application up and running</p>
```

Comment:
Unit tests in `landing.controller.test.ts` will need to be updated to match the new content. Also think we need to add coverage for clicking the start button / `post` method.